.evg.yml

```
@@ -230,6 +230,39 @@ tasks:
       - func: "publish maven"
       - func: "trace artifacts"
 
+  # Publishes the release artifacts to S3
+  - name: publish-s3
+    allowed_requesters: ["ad_hoc", "github_tag", "patch"]
+    depends_on:
+      - name: "build"
+        variant: "release"
+      - name: "test-smoke"
+        variant: "release"
+    commands:
+      - *assume_role_cmd
+      - command: s3.get
+        params:
+          working_dir: mongo-jdbc-driver
+          <<: *evg_bucket_config
+          local_file: mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
+          remote_file: ${S3_ARTIFACTS_DIR}/mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
+      - command: s3.get
+        params:
+          working_dir: mongo-jdbc-driver
+          <<: *evg_bucket_config
+          local_file: mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
```

```suggestion
          local_file: mongodb-jdbc-eap-${MDBJDBC_VER}.jar
```
Small typo so we don't overwrite the previous `s3.get`